nibe/data/extensions.json

```diff
@@ -105,5 +105,124 @@
         }
       }
     }
+  },
+  {
+    "description": "M12676EN-1 - S1155/S1255 ",
+    "files": ["s1155_s1255.json"],
+    "data": {
+      "31101": {
+        "mappings": {
+          "0": "Off",
+          "1": "On"
+        }
+      }
+    }
+  },
+  {
+    "description": "M12676EN-1 - SMOS40 Mappings",
+    "files": ["smos40.json"],
+    "data": {
+      "31067": {
+        "mappings": {
+          "0": "Off",
+          "1": "On"
+        }
+      }
+    }
+  },
+  {
+    "description": "M12676EN-1 - F2120 slave 1",
+    "files": ["smos40.json"],
+    "data": {
+      "31806": {
+        "mappings": {
+          "0": "Off",
+          "1": "Active",
+          "2": "Passive"
+        }
+      }
+    }
+  },
+  {
+    "description": "M12676EN-1 - F2120 slave 2",
+    "files": ["smos40.json"],
+    "data": {
+      "31792": {
+        "mappings": {
+          "0": "Off",
+          "1": "Active",
+          "2": "Passive"
+        }
+      }
+    }
+  },
+  {
+    "description": "M12676EN-1 - F2120 slave 3",
+    "files": ["smos40.json"],
+    "data": {
+      "31778": {
+        "mappings": {
+          "0": "Off",
+          "1": "Active",
+          "2": "Passive"
+        }
+      }
+    }
+  },
+  {
+    "description": "M12676EN-1 - F2120 slave 4",
+    "files": [],
```

Comment:
empty array means do not add anywhere?

Reply:
Yes. Seems we have no export with multiple slaves yet